home/docs/help/nvidia.md

```diff
@@ -0,0 +1,36 @@
+---
+id: nvidia  
+title: NVIDIA Monitoring  
+sidebar_label: NVIDIA Monitoring  
+keywords: [Open Source Monitoring System, NVIDIA Monitoring]
+---
+
+> Collect and monitor general performance metrics of NVIDIA operating systems.
```

Comment:
Hi, do we need to explain here the environment that needs to be installed in advance, such as nvidia-smi

Reply:
modified